flow/scene_update_context.h

```diff
@@ -100,7 +103,10 @@ class SceneUpdateContext {
           const SkRRect& rrect,
           SkColor color,
           float elevation);
-    ~Frame();
+
+    ~Frame() = default;
+
```

Comment:
Could you add a comment for what Finalize does?

Reply:
Done.